### DISCUSSION TITLE
Bug: The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>.

### DISCUSSION THREAD

**Discussion by AniBlox** [eritislami/evobot#433] [Q&A]
**Describe the bug**
when I use play song name it gives me this error
* Node.js version: v14.15.1

**Additional information & screenshots**
![image](https://user-images.githubusercontent.com/72016134/101128497-de300e00-3610-11eb-8f8f-079bf6cc708b.png)

pls help me fix this this only happens when i deploy to heroku

**Comment by booleans-oss** [comment]
Looks like you've exceeded the quota: [Youtube's quota](https://developers.google.com/youtube/v3/getting-started#quota). Briefly, it means that you have done too many search requests or basic requests at all in a same day.

What you can do, but I won't recommend it because it is kind of by-passing their quota, is to use another Google Account with Youtube v3 API. In that way, your quota will start again from 0 but I repeat, I kindly don't suggest to do it that way
